graphjet-core/src/main/java/com/twitter/graphjet/algorithms/counting/TopSecondDegreeByCount.java

```diff
@@ -74,14 +74,23 @@ public TopSecondDegreeByCount(
   }
 
   /**
+   * Returns whether we should update the node info, according the request requirements.
+   * It is used to filter information of unwanted edges from being aggregated.
+   * @param request       is the request given by the requester
+   * @param edgeIterator  is the iterator being used to iterate node's edges.
+   *                      It carries information such as last engagement time of the current edge
```

Comment:
engagement time of the current edge -> the engagement time of the current edge. 

for an individual edge, there is only one engagement time.